tests/integration/test_impersonation.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from tenable_io.api.users import UserCreateRequest
+
+from tests.base import BaseTest
+from tests.config import TenableIOTestConfig
+
+
+class TestImpersonation(BaseTest):
+
+    @pytest.fixture(scope='class')
+    def user(self, app, client):
+        new_user = client.users_api.create(UserCreateRequest(
+            username=app.session_name(u'test_impersonation%%s@%s' % TenableIOTestConfig.get('users_domain_name')),
+            name='test_impersonation',
+            password='test_impersonation',
+            permissions='16',
+            type='local'
+        ))
+        user_list = client.users_api.get(new_user)
+        yield user_list.username
```

Comment:
This is not an `user_list`, should be an `user`. Also let's just return the `user` full object instead of just `username`.